src/types/user.ts

```diff
@@ -111,7 +111,8 @@ export function toAppUser(
   return {
     bpd_token: bpdToken,
     created_at: new Date().getTime(),
-    date_of_birth: formatDate(from.dateOfBirth),
+    date_of_birth:
+      from.dateOfBirth !== undefined ? formatDate(from.dateOfBirth) : undefined,
```

Comment:
Maybe it's just nitpicking, but I prefer to rely on well-tested functions to ensure nullity
```suggestion
    date_of_birth: fromNullable(from.dateOfBirth)
      .map(formatDate)
      .toUndefined(),
```

Reply:
fixed in 14ba994